index.bs

```diff
@@ -513,7 +513,7 @@ Common Encryption {#CommonEncryption}
 
 When the protected scheme <code>[=cenc=]</code> is used, samples SHALL be protected using subsample encryption and SHALL NOT use pattern encryption.
 
-When the protected scheme <code>[=cbcs=]</code> is used, samples SHALL be protected using subsample encryption and SHALL use pattern encryption. In particular, pattern encryption with <code>crypt_byte_block</code> = 1 and <code>skip_byte_block</code> = 0 SHALL be supported.
+When the protected scheme <code>[=cbcs=]</code> is used, samples SHALL be protected using subsample encryption and SHALL use pattern encryption. [[!CENC]] recommends that the Pattern Block length be 10, i.e. <code>crypt_byte_block + skip_byte_block = 10</code>. All combinations such that this is true SHALL be supported, in particular, <code>crypt_byte_block</code> = 10 and <code>skip_byte_block</code> = 0.
```

Comment:
I still don't think it is a good idea to support non pattern encryption in pattern encryption scheme.

Anyway, this leaves the tail bytes, up to 159 bytes, unencrypted. Is that desirable?

Reply:
> Anyway, this leaves the tail bytes, up to 159 bytes, unencrypted. Is that desirable?

Can you clarify why you think so? CENC says:
> If the last Block pattern in a Subsample is incomplete, the partial pattern SHALL be followed until
truncated by the BytesOfProtectedData size and any partial crypt_byte_block SHALL
remain unencrypted.

I agree the wording is not perfect. I interpret as saying the last partial block (i.e. <16 bytes) remains unencrypted, and anyway partial blocks are not allowed.

Reply:
I read the spec again. Yes, you are right. I mis-interpreted the spec previously.